server.js

```diff
@@ -56,72 +57,37 @@ app.get('/itemInformation/:itemId', (req, res) => {
 
   if (itemId.includes('array')) {
     const itemsInArray = itemId.substring(5);
-    const itemIds = itemsInArray.split(',');
-    const invalidId = false;
-
-    for (var i = 0; i < itemIds.length; i++) {
-      if (itemIds[i] < 100 || itemIds[i] > 199) {
-        res.status(404).send('Invalid itemId');
-        invalidId = true;
-        break;
-      }
-    }
-
-    if (!invalidId) {
-      db.getTitlesAndBrands(itemIds)
-        .then((data) => {
-          res.send(data);
-        })
-        .catch((err) => {
-          res.status(404).send('error in getTitlesAndBrands: ', err);
-        });
-    }
-  } else if (itemId < 100 || itemId > 199) {
-    console.log(itemId);
-    res.status(404).send('Invalid itemId');
-  } else {
-    db.getTitleAndBrand(itemId)
+    const itemIds = itemsInArray.split(',').map((id) => Number(id));
+    postgres
+      .getMultipleTitlesAndDecsriptions(...itemIds)
       .then((data) => {
-        console.log('success getting title and brand');
-        res.send(data[0]);
+        res.status(200).send(data);
       })
       .catch((err) => {
-        res.status(500).send(err);
-        console.log('error in getTitleAndBrand: ', err);
+        res
+          .status(404)
+          .send(
+            'One of the ids submitted was invalid or the input request was formatted imporperly!'
+          );
+      });
+  } else {
+    postgres
+      .getOneTitleAndDescription(Number(itemId))
+      .then((response) => {
+        res.status(200).send(response);
+      })
+      .catch((err) => {
+        res.status(404).send('The id searched for was invalid!');
       });
   }
 });
 
 //get full description object for an item
 app.get('/descriptionObject/:itemId', (req, res) => {
   const itemId = req.params.itemId;
-
-  db.getDescriptionObject(itemId)
-    .then((data) => {
-      console.log('success getting descriptionObj');
-
-      var formattedData = {
-        description: {
-          title: data[0].title,
-          description: data[0].description,
-          SKU: data[0].SKU,
-          primaryBrand: data[0].primaryBrand,
-          daysToShip: data[0].daysToShip,
-        },
-        directions: {
-          directions: data[0].directions,
-        },
-        attributes: {
-          primaryColor: data[0].primaryColor,
-          material: data[0].material,
-          length: data[0].length,
-          width: data[0].width,
-        },
-        details: {
-          additionalDetails: data[0].additionalDetails,
-        },
-      };
-
+  postgres
+    .getDataForSpecifiedId(itemId)
+    .then((formattedData) => {
       res.send(formattedData);
```

Comment:
is there a reason for not setting the status code here?

Reply:
No there isnt. I should add that.